### DISCUSSION TITLE
Can't Create in file inside my file.

### DISCUSSION THREAD

**Discussion by WilliamNasr** [community/community#17747] [GitHub Education]
Hello! I need to create a file _data/WilliamNasr.md inside _data/WilliamNasr , but unfortunately there is no add new file, only "go to file" section is appearing. Anyone can help?
![Screenshot (307)](https://user-images.githubusercontent.com/77879406/169638491-cd1f60eb-89e8-48c4-ac79-266118a3338b.png)

**Comment by msMohit20** [comment]
I am having the same kind of denial

**Comment by htcooper** [comment]
You can add a folder by putting a / after the folder name, and then follow it with the file name - meaning, you create the new folder and the new file at the same time. 
- go to add file > create new file
- then type WilliamNasr/WilliamNasr.md in the box where it says "name your file". As soon as you type the "/" you'll see the first part turn into a folder name.

**Comment by msMohit20** [comment]
okay bu my username start with small latter so that i can't.
